### PR TITLE
[Metricbeat] Add CouchDB Dashboard

### PR DESCRIPTION
**Pull request by sorantis**
Partially addresses #10594.

This CouchDB dashboard visualizes the following metrics:

```yaml

  - name: couchdb.server.httpd.view_reads
    description: Number of view reads.
  - name: couchdb.server.httpd.temporary_view_reads
    description: Number of temporary view reads.

  - name: couchdb.server.couchdb.open_databases
    description: Number of open databases.

  - name: couchdb.server.httpd_status_codes.*
    description: HTTP status code response rates (200, 201, 202, 400, 401, 403, 404, 405, 409, 500).

  - name: couchdb.server.httpd_request_methods.*
    description: HTTP request method rates (GET, POST, PUT, DELETE, HEAD, COPY).

  - name: couchdb.server.couchdb.database_reads
    description: Number of times a document was read from a database.
  - name: couchdb.server.couchdb.database_writes
    description: Number of times a database was changed.

  - name: couchdb.server.httpd.view_reads
    description: Number of view reads.
  - name: couchdb.server.httpd.temporary_view_reads
    description: Number of temporary view reads.

  - name: couchdb.server.couchdb.auth_cache_hits
    description: Number of authentication cache hits.
  - name: couchdb.server.couchdb.auth_cache_misses
    description: Number of authentication cache misses.
```
Dashboard snapshot:

metricbeat-couchdb-overview

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -37,6 +37,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Metricbeat*
 
+- Add new dashboard for CouchDB database
```
Comment (jsoriano) [inline CHANGELOG.next.asciidoc]:
```suggestion
- Add new dashboard for CouchDB database {pull}13198[13198]
```

File: metricbeat/module/couchdb/_meta/kibana/7/dashboard/Metricbeat-couchdb-overview.json [hunk 2]
```diff
@@ -0,0 +1,1339 @@
...
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "[Metricbeat CouchDB] HTTP Status Codes",
```
Comment (jsoriano) [inline metricbeat/module/couchdb/_meta/kibana/7/dashboard/Metricbeat-couchdb-overview.json]: Visualization titles should have the module tag at the end
```suggestion
        "title": "HTTP Status Codes [Metricbeat CouchDB]",
```

And to make them nicer, once placed in the dashboard, you can edit and select different titles there.

Comment (sorantis) [reply metricbeat/module/couchdb/_meta/kibana/7/dashboard/Metricbeat-couchdb-overview.json]: done

File: CHANGELOG.next.asciidoc (hunk 1 above)
Comment (sorantis) [reply CHANGELOG.next.asciidoc]: updated

**Comment by sorantis** [comment]
> Are open databases the number of open connections? I think it is useful to see the number of connections over time.

Yes. Changed to a TS graph.

File: metricbeat/module/couchdb/_meta/kibana/7/dashboard/Metricbeat-couchdb-overview.json [hunk 3]
```diff
@@ -0,0 +1,1347 @@
+{
+  "objects": [
+    {
+      "attributes": {
+        "description": "This CouchDB dashboard visualizes the most important CouchDB server metrics.\n\n",
```
Comment (jsoriano) [inline metricbeat/module/couchdb/_meta/kibana/7/dashboard/Metricbeat-couchdb-overview.json]: Why are you adding the newlines at the end `\n\n`?

**Pull request merged**